_posts/2020-11-09-pulling-most-recent-files.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title:  "Pull most recent files in R"
+date:   2020-11-09
+categories: R base package
+---
+# Pulling most recent files in R
+ ###### by Joel Christian Andrade
+
+Many coding pipelines involve taking raw data files, formulating code and eventually working with a few, if not many, output files.
+
+Take these files in the directory `/raw_data` below, downloaded from publicly available Apple mobility database tracking movement.
+
+```
+applemobilitytrends_2020-08-29.csv
+applemobilitytrends_2020-09-12.csv
+applemobilitytrends_2020-09-18.csv
+applemobilitytrends_2020-09-25.csv
+applemobilitytrends_2020-09-30.csv
+applemobilitytrends_2020-10-02.csv
+applemobilitytrends_2020-11-08.csv
+```
+
+While retroactive data analysis is useful, the most recent file will include all data from previous files.  Thus, to retrieve the most up to date information while using R, we must use some functions which are pre-loaded into R.
+
```

Comment:
@jsandrad consider adding another header here...something like `Useful R Functions` or `R functions we will be using`